hello-1/conanfile.py

```diff
@@ -1,18 +1,21 @@
-from conans import ConanFile
+from conans import ConanFile, tools
 
 class HelloConan(ConanFile):
     name = "hello"
     version = "0.1"
     exports = "*"
     generators = "qmake"
+    settings = "os", "compiler", "arch", "build_type"
 
     def build(self):
-        self.run('qmake')
-        self.run("make")
+        self.run('qmake -tp vc')
+        vcvars = tools.vcvars_command(self.settings)
+        self.run("%s && msbuild hello.vcxproj /p:configuration=debug" % vcvars)
```

Comment:
The ``debug`` configuration is hardcoded into the recipe, this is wrong. 
Typically, the ``settings.build_type`` will be used to replace it, and use the correct setting.

Also, a typical, generic ``build()`` would be conditional to the settings, like:

```python
def build(self):
    if self.settings.os == "Windows":
        self.run("qmake -tp vc")
        ...
    else:
        self.run("qmake (other args?)")
```